### ISSUE TITLE
Some confusions about lime-Tutorial%20-%20images

### ISSUE THREAD

**Issue by ChengruiMeng** [marcotcr/lime#242]
Hello，I'm a rookie of ML. I am honored to ask you.
I have read your paper and try to run your Tutorial. During this process，there are two problem i met :
        1.I can't understand why there are 1001 classes? Shouldn’t it be 1000?
`import os
with slim.arg_scope(inception.inception_v3_arg_scope()):
    logits, _ = inception.inception_v3(processed_images, num_classes=1001, is_training=False)
probabilities = tf.nn.softmax(logits)`
       2.I try to run it for each synset to see what accuracy rate in this models by using LIME explanation，so I download ILSVRC2012_img_val to run this example. But I find the class index can not match in your experience.For example，the "English setter" ’s index in imageNET is “1”，but in your experience is shown as "213". 
`178 Scottish deerhound, deerhound 0.00051360053
214 Irish setter, red setter 0.0007739145
215 Gordon setter 0.0014154159
170 borzoi, Russian wolfhound 0.0017604019
213 English setter 0.9593616
`
Why is there a difference?
I‘m looking forward to your reply.

**Comment by ChengruiMeng** [comment]
all right i find the problems. sorry to pull such a stupid question.:)
